react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle } from '@fortawesome/free-regular-svg-icons';
+import { faPlay, faPause } from '@fortawesome/free-solid-svg-icons';
+export default function Stopwatch({ into }) {
+  const [begin, realTime] = useState(0);
+  const [timerOn, setTime] = useState(false);
+  const [clearIntervalID, setIntervalID] = useState();
+
+  function onStart() {
+    setTime(true);
+    const interval = setInterval(() => {
+      realTime(begin => begin + 1);
+    }, 1000);
+    setIntervalID(interval);
+  }
+
+  function handlePause() {
+    clearInterval(clearIntervalID);
```

Comment:
Since you've cleared the interval, `clearIntervalID` is no longer valid. You should unset it.